spec/units/models/unsubscribe_link_spec.rb

```diff
@@ -9,7 +9,7 @@
   describe "#url" do
     it "returns an unsubscribe url for the subscription" do
       expect(unsubscribe_link.url).to eq(
-        "http://www.dev.gov.uk/email/unsubscribe/1234?title=dave%20crocker%20%26%20friends"
+        "http://www.dev.gov.uk/email/unsubscribe/1234?title=dave+crocker+%26+friends"
```

Comment:
I don't remember the details, but I think we need to escape the `+` symbols otherwise the unsubscribe links are rendered correctly in the email. That's why we used `URI.encode(value, /\W/)`. Maybe @gpeng can remember more?

Reply:
Ah... I did wonder if that was the case. `+`s are a bit of a weird thing, and I couldn't find a suitable replacement for `URI.encode` which supported the second argument.

Reply:
Yeah... I remembered us having trouble with this but I couldn't remember what the problem was. I thought we were just trying to not have `+` urls. Let's deploy it to integration and check these manually before we merge.

Reply:
Yep, turns out the pluses get turned into `%2B` so the URLs don't work.

Reply:
Looks like we can use https://stackoverflow.com/a/39209414 instead.

Reply:
@gpeng @tuzz Have updated the PR to use `ERB::Util.url_encode`.